lib/OpenAPI/Client.pm

```diff
@@ -46,6 +46,7 @@ sub new {
   $self->ua->transactor->name('Mojo-OpenAPI (Perl)') unless $self->{ua};
 
   if (my $app = delete $self->{app}) {
+    warn "[$class] Found supplied app\n" if DEBUG;
```

Comment:
Same here... Isn't that message implicit? When don't you know that `app` is supplied?